README.rst

```diff
@@ -75,13 +75,15 @@ Usage
     -p, --cog-profile [jpeg|webp|zstd|lzw|deflate|packbits|raw] CloudOptimized GeoTIFF profile (default: deflate).
     --nodata NUMBER|nan             Set nodata masking values for input dataset.
     --add-mask                      Force output dataset creation with an internal mask (convert alpha band or nodata to mask).
-    --overview-level INTEGER        Overview level (if not provided, appropriate overview level will be selected until the
-                                    smallest overview is smaller than the internal block size).
+    --overview-level INTEGER        Overview level (if not provided, appropriate overview level will be selected until the smallest overview is smaller than the value of the internal blocksize).
     --overview-resampling [nearest|bilinear|cubic|cubic_spline|lanczos|average|mode|gauss] Resampling algorithm.
     --overview-blocksize TEXT       Overview's internal tile size (default defined by GDAL_TIFF_OVR_BLOCKSIZE env or 128)
+    -w, --web-optimized             Create COGEO optimized for Web.
+    --latitude-correction           Apply latitude correction to ensure max zoom equality for dataset accross different latitudes.
```

Comment:
@vincentsarago it's a little bit unclear to me which choice latitude correction refers to:
1. _uncorrection_ (== global maxzoom regardless of latitude)
2. Maxzoom based on actual on-the-ground resolution

Reply:
yes this was also unclear to me, when passing `--latitude-correction` it will adjust the resolution to match the resolution at equator (so global maxzoom regardless of latitude).

I'm not sure what default it should be then, `uncorrection` seems to be a better choice but this is a personal opinion

Reply:
🤔 rather than correct / not correct (which comes with problems of semantic interpretation) what about `--latitude-adjustment` where `True` indicates that it adjusts for latitude, and `False` indicates that it doesn't and simply uses mercator pixels. 

Or, it could be `--mercator-meters` which is clear (use mercator meters for calculation) but possibly obscured to those that aren't familiar with the fact that a mercator meter only ~= an on the ground meter near the equator?

Reply:
👍 
the `--mercator-meters|--src-crs-meters` seems a bit complex even if I understand if better. 

I wonder if we could do something like `--latitude-adjustment|--global-maxzoom`, I'm not really happy with this semantic either tbh